README.rst

```diff
@@ -230,19 +230,26 @@ metric registry data directly to SignalFx.
 
 .. code:: python
 
-    from pyformance import count_calls, gauge
+    from signalfx.pyformance import count_calls, gauge, count_calls_with_dims
```

Comment:
Might as well import `SignalFxReporter` along with it so you don't need the separate `import` line below (and change line 246)